crates/bevy_animation_rig/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "bevy_animation_rig"
```

Comment:
IMO this should be the start of a general `bevy_animation` crate instead.

Reply:
The original `bevy_animation` is meant to animate the component properties. So I created this crate to separate the "skinning" from the "animation".